harness/harness.sh

```
@@ -0,0 +1,134 @@
+STATE_FILE="$HARNESS_DIR/task_state.json"
+TEMP_FILE="$STATE_FILE.next"
+
+# 색상 정의
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[1;33m'
+CYAN='\033[0;36m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+cd "$PROJECT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)